design-documents/graph-ql/coverage/CartPromotions.graphqls

```diff
@@ -0,0 +1,53 @@
+type Cart{
+
+    /// Deprecated Feilds
+    applied_coupon: applied_coupon @deprecated(reason: "Use CartPromotions.coupons instead ")                               
```

Comment:
I'd suggest providing new field (not type) in the deprecated message. In this case it's probably something like `prices.discounts.coupons`